Example_README.md

```diff
@@ -9,6 +9,14 @@ Provide a short description of your template and use case here. Try to keep it t
 
 ![Example Dashboard Screenshot](Example_Screenshot.png)
 
+### Quick Install
+
+If you have your `INFLUX_HOST`, `INFLUX_ORG` and `INFLUX_TOKEN` environment variables setup, you can install this template with:
```

Comment:
I believe the config should have it as well, env vars aren't required anymore. Something to consider 🤷